File: fix_parens.py
import sys

fixes = [
    (r'd:\XBOX\qcare\hospital_queue_app\lib\screens\token_status_screen.dart', 224),
    (r'd:\XBOX\qcare\hospital_queue_app\lib\screens\my_appointments_screen.dart', 109),
    (r'd:\XBOX\qcare\hospital_queue_app\lib\screens\book_appointment_screen.dart', 367),
]

for path, bad_line in fixes:
    with open(path, 'r', encoding='utf-8-sig') as f:
        lines = f.readlines()
    removed = lines[bad_line - 1].rstrip()
    del lines[bad_line - 1]
    with open(path, 'w', encoding='utf-8', newline='') as f:
        f.writelines(lines)
    print(f'OK: removed line {bad_line} from {path.split(chr(92))[-1]}  [{repr(removed)}]')

print('Done.')

File: fix_parens.js
const fs = require('fs');

const fixes = [
    { file: 'd:\\XBOX\\qcare\\hospital_queue_app\\lib\\screens\\token_status_screen.dart', line: 224 },
    { file: 'd:\\XBOX\\qcare\\hospital_queue_app\\lib\\screens\\my_appointments_screen.dart', line: 109 },
    { file: 'd:\\XBOX\\qcare\\hospital_queue_app\\lib\\screens\\book_appointment_screen.dart', line: 367 },
];

for (const { file, line } of fixes) {
    const content = fs.readFileSync(file, 'utf8');
    const lines = content.split(/\r?\n/);
    const removed = lines[line - 1];
    lines.splice(line - 1, 1);
    fs.writeFileSync(file, lines.join('\r\n'), 'utf8');
    const name = file.split('\\').pop();
    console.log(`OK: removed line ${line} from ${name}: [${removed.trim()}]`);
}
console.log('Done.');
